manifests/t/ThePandemoniumInstitute/BloodontheClocktowerOnline/3.23.1/ThePandemoniumInstitute.BloodontheClocktowerOnline.locale.en-US.yaml

```
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: ThePandemoniumInstitute.BloodontheClocktowerOnline
+PackageVersion: 3.23.1
+PackageLocale: en-US
+Publisher: The Pandemonium Institute

```

```suggestion
Publisher: Steffen Baumgart
```
This should match control panel